bin/csv_from_oai.rb

```diff
@@ -0,0 +1,80 @@
+require 'csv'
+require 'oai'
+require 'byebug'
```

Comment:
Before we merge this, we'll want to remove the `require "byebug"; it is not best practices to keep a require of that library in code that might run in production.

Reply:
Removed